docs/spec/v2/log.json

```diff
@@ -9,6 +9,169 @@
         "integer"
       ]
     },
+    "dataset": {
+      "description": "Dataset identifies the source which originated the log line.",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 1024
+    },
+    "ecslogerrorfields": {
```

Comment:
I wonder if we should just not produce JSON Schema for logs for now? IIRC that's what @felixbarny suggested earlier. Otherwise we should probably ensure the embedded structs don't affect the schema.

Reply:
> I wonder if we should just not produce JSON Schema for logs for now?

How are the generated specs utilized? FWIW, I was planning on fixing the embedded struct handling in the schema generation code but if we don't have a use case for the schema or if it is not required for the logs then removing it sounds good to me.

Reply:
At least some of the agents use the schema for payload validation in tests. We used to use it in APM Server, before we generated the validation code. Let's drop it for now and come back to it if agents want to start validate logs payloads.

Reply:
Thanks @axw, I have removed the JSON spec generation for now.